repos/arcade.proj

```diff
@@ -52,10 +52,6 @@
     <EnvironmentVariables Include="DotNetUseShippingVersions=false" />
   </ItemGroup>
 
-  <ItemGroup>
-    <RepositoryReference Include="newtonsoft-json" />
```

Comment:
We need to actually enforce Arcade building before Newtonsoft.Json, right?  We should add a `<RepositoryReference Include="arcade" />` to newtonsoft-json.proj with a comment that it's not a real dependency and for ordering purposes only.